Platform/MeteorlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c

```diff
@@ -434,7 +434,7 @@ UpdateFspConfig (
   Fspmcfg->PchHdaEnable                                         = 0x1;
   Fspmcfg->DmiMaxLinkSpeed                                      = 0x1;
   Fspmcfg->DmiAspm                                              = 0x4;
-  Fspmcfg->Gen3LtcoEnable                                       = 0x1;
+  Fspmcfg->Gen3LtcoEnable                                       = 0x0;
```

Comment:
Update this in cfg data.

Reply:
fixed, Updated value in mtlp dlt file instead